### PR TITLE
Added procedural steps for "Following People on GitHub"

### LINKED ISSUE #4856: Update "Following people" article, part two
This issue was reported/written by @ethanpalm. ✨ 

### What article on docs.github.com is affected?

"[Following people](https://docs.github.com/en/github/getting-started-with-github/following-people)"

### What part(s) of the article would you like to see updated?

The article "[Following people](https://docs.github.com/en/github/getting-started-with-github/following-people)" doesn't follow our current [content style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md) and is missing information. This article can be improved by adding an "About followers on GitHub" section, rewriting the existing content for following someone as procedural steps in new sections, and adding a new section "Viewing followed users and followers."

Audience: New users of the followers features on GitHub.

### Content to update

"[Following people](https://docs.github.com/en/github/getting-started-with-github/following-people)"
- [x] Add "Following a user" section
  - [x] In this section, rewrite the existing information on following a user as procedural steps
  - [x] Add screenshots if needed
- [x] Add "Unfollowing a user" section
  - [x] In this section, rewrite the existing information for following a user as procedural steps
  - [x] Add screenshots if needed

For more information on writing procedural steps, see the [content style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#procedural-steps).

### Additional information
This issue is for a special project and is not currently available for contributors to work on.

Companion issue: https://github.com/github/docs/issues/4855

### PR DESCRIPTION
**Pull request by ramyaparimi**
### Why:

Closes https://github.com/github/docs/issues/4856
✨ [rendered page](http://localhost:4000/en/get-started/exploring-projects-on-github/following-people) ✨ 

### What's being changed:
The article "[Following people](https://docs.github.com/en/github/getting-started-with-github/following-people)" doesn't follow our current [content style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md) and is missing information. This article can be improved by adding an "About followers on GitHub" section, rewriting the existing content for following someone as procedural steps in new sections, and adding a new section "Viewing followed users and followers."

Audience: New users of the followers features on GitHub.

### Content updated

"[Following people](https://docs.github.com/en/github/getting-started-with-github/following-people)"
- [x] Add "Following a user" section
  - [x] In this section, rewrite the existing information on following a user as procedural steps
  - [x] Add screenshots if needed
- [x] Add "Unfollowing a user" section
  - [x] In this section, rewrite the existing information for following a user as procedural steps
  - [x] Add screenshots if needed

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [x] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:18]: Small typo:

```suggestion
When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed{% endif %}.
```

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:24]: Suggest this might be clearer as:

```suggestion
1. Click a user image to display that user's profile.
```

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:26]: The link to the image can be defined just from `/assets` and it should probably have a slightly more informative description. I think the image also needs to be indented by two spaces to ensure that the numbered steps are correctly displayed.

```suggestion
  ![User image](/assets/images/help/profile/user-profile-image.png) 
```

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:28]: There's a lot to remember, but we don't usually mention that it's a button, see advice in the style guide: https://github.com/github/docs/blob/main/contributing/content-style-guide.md#buttons.

```suggestion
1. Click **Follow** under their profile image.
```

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:40]: As before:

```suggestion
1.  Click a user image to display that user's profile.

  ![User image](/assets/images/help/profile/user-profile-image.png) 

1.  Click **Unfollow** under their profile image.

  ![User profile button](/assets/images/help/profile/user-profile-unfollow-button.png) 
```

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:30]: Indenting the image so that it appears within the numbered steps.
```suggestion
  ![Follow user button](/assets/images/help/profile/follow-user-button.png)
```

**Review by felicitymay — COMMENTED** [review]
Thanks for updating the docs to bring this article into line with our style guide 💖 

My comments are all for small things. Generally your changes look good in the preview: [Following people](https://docs-8900--following-people.herokuapp.com/en/get-started/exploring-projects-on-github/following-people). Once you've merged my suggested changes, you should check whether the numbered steps are correctly numbered. If they are still all showing as "1", you'll need to remove the blank lines between steps and images in each procedure, but this might not be necessary.

I notice that as well as addressing the points in https://github.com/github/docs/issues/4856, you've also addressed some of the changes mentioned in: https://github.com/github/docs/issues/4855. You might want to check the boxes for the parts of the work that are included in this PR.

File: .eslintcache
Comment (ramyaparimi) [reply .eslintcache:1]: That must have happened when I pulled the changes in main. Please let me know if I should be reverting this change.

**Comment by ramyaparimi** [comment]
@felicitymay Thank you so much for such a speedy review 💖

File: content/get-started/exploring-projects-on-github/following-people.md
Comment (felicitymay) [inline content/get-started/exploring-projects-on-github/following-people.md:18]: Sorry, just realized that we still have `..` in the GHES and GHAE version of the docs...

```suggestion
When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed.{% endif %}
```

File: .eslintcache
Comment (felicitymay) [reply .eslintcache:1]: It looks as if this file was added by https://github.com/github/docs/pull/8900/commits/62b3f94b33dd071717df486c3669b3f271b9fe34. It definitely shouldn't be included in your PR as it's listed in the `.gitignore` file for this repository: https://github.com/github/docs/blob/main/.gitignore.

The fact that you were able to commit this file suggests that there's something odd going on in your local git checkout of the `docs` repository. Removing a file that's included in a `.gitignore` file from a pull request can be tricky (you might need to comment out this filename in the `.gitignore` file before you can create a commit that removes this file, then remove the comment).

**Review by felicitymay — COMMENTED** [review]
Thanks for accepting the changes @ramyaparimi 

Everything looks good now and this is ready to merge once you've removed the `.eslintcache` file from the PR.

File: .eslintcache
Comment (ramyaparimi) [reply .eslintcache]: Okay, thank you for explaining how to revert the change 💖   I now cant see in the list of files changed 🐬

**Review by felicitymay — APPROVED** [review]
This looks ready to merge now that you've successfully removed the inadvertently included file 💖

**Pull request merged**
